.github/workflows/cre-workflow-load-tests.yaml

```diff
@@ -0,0 +1,141 @@
+    runs-on: ubuntu-latest
+    environment: "integration"
+    timeout-minutes: 20
+    permissions:
+      contents: read
+      id-token: write
+    if: github.event_name == 'workflow_dispatch' || github.event_name == 'workflow_call' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'run-cre-workflow-don-test'))
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'pull_request' && github.sha || inputs.chainlink_version }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: core/scripts/go.mod
+
+      # We need to login to ECR to allow the test to pull the Job Distributor and Chainlink images
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a # v4.0.1
+        with:
+          aws-region: ${{ secrets.QA_AWS_REGION }}
+          role-to-assume: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+          role-duration-seconds: 1800
+          mask-aws-account-id: true
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076 # v2.0.1
+        with:
+          registries: ${{ format('{0},{1}', secrets.QA_AWS_ACCOUNT_NUMBER, secrets.AWS_ACCOUNT_ID_PROD) }}
+        env:
+          AWS_REGION: ${{ secrets.QA_AWS_REGION }}
+
+      - name: Set date variable
+        id: set-date
+        if:  ${{ github.event_name == 'pull_request' }}
+        run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
+
+      - name: Overwrite chainlink version in TOML config
+        shell: bash
+        run: |
+          cat > system-tests/tests/load/cre/ci-override.toml<< EOF
+          [jd]
+          image = "injected-at-runtime"
+          
+          [[nodesets]]
+          nodes = 6
+          override_mode = "all"
+          http_port_range_start = 10100
+          name = "workflow"
+          [nodesets.db]
+          image = "postgres:12.0"
+          port = 13000
+          volume_name = "workflow_volume"
+          [[nodesets.node_specs]]
+          [nodesets.node_specs.node]
+          image = "injected-at-runtime"
+          
+          [[nodesets]]
+          nodes = 3
+          override_mode = "each"          
+          http_port_range_start = 10200  
+          name = "capabilities"
+          [nodesets.db]
+          image = "postgres:12.0"
+          port = 13200
+          [[nodesets.node_specs]]
+          [nodesets.node_specs.node]
+          image = "injected-at-runtime"
+          custom_ports = ["13401:7777"]
+          [[nodesets.node_specs]]
+          [nodesets.node_specs.node]
+          image = "injected-at-runtime"
+          custom_ports = ["13402:7777"]
+          [[nodesets.node_specs]]
+          [nodesets.node_specs.node]
+          image = "injected-at-runtime"
+          custom_ports = ["13403:7777"]
```

Comment:
wont' the config we have saved in repo work? is the override necessary?

Reply:
Added a CI config file and removed the overwrites